drivers/soundwire/bus.c

```diff
@@ -1272,6 +1272,15 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 			sdw_write_no_pm(slave, SDW_SCP_INT1, SDW_SCP_INT1_BUS_CLASH);
 		}
 	}
+	if ((slave->bus->prop.quirks & SDW_MASTER_QUIRKS_CLEAR_INITIAL_PARITY) &&
+	    !(slave->prop.quirks & SDW_SLAVE_QUIRKS_INVALID_INITIAL_PARITY)) {
```

Comment:
Do we need test `SDW_SLAVE_QUIRKS_INVALID_INITIAL_PARITY`?

Reply:
yes, it was on-purpose 

'This patch suggests an additional bus-level quirk for parity, which is
only applied when the codec device is not known to have an issue with
parity.'

If we make this unconditional, then the SDW_SLAVE_QUIRKS_INVALID_INITIAL_PARITY is useless and will never be tested for.